service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/SocialWorkOtherControllerTest.java

```diff
@@ -0,0 +1,127 @@
+import uk.gov.hmcts.reform.authorisation.ServiceAuthorisationApi;
+import uk.gov.hmcts.reform.ccd.client.CaseAccessApi;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.idam.client.IdamApi;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@ActiveProfiles("integration-test")
+@WebMvcTest(SocialWorkOtherSubmissionController.class)
+@OverrideAutoConfiguration(enabled = true)
+public class SocialWorkOtherControllerTest {
+    private static final String AUTH_TOKEN = "Bearer token";
+    private static final String USER_ID = "10";
+    private static final String CASE_ID = "1";
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private ServiceAuthorisationApi serviceAuthorisationApi;
+
+    @MockBean
+    private IdamApi idamApi;
+
+    @MockBean
+    private CaseAccessApi caseAccessApi;
+
+    @Test
+    void shouldReturnWithErrorsIfDocumentTitleIsNotPopulated() throws Exception {
+        CallbackRequest request = createCallbackRequest("");
+
+        MvcResult response = performResponseCallBack(request);
+
+        AboutToStartOrSubmitCallbackResponse callbackResponse = MAPPER.readValue(response.getResponse()
+            .getContentAsByteArray(), AboutToStartOrSubmitCallbackResponse.class);
+
+        assertThat(callbackResponse.getErrors()).contains("You must give additional document 1 a name.");
+    }
+
+    @Test
+    void shouldReturnWithErrorsIfMultipleDocumentTitlesHaveNotBeenProvided() throws Exception {
+
+        CallbackRequest request = CallbackRequest.builder().caseDetails(CaseDetails.builder()
+            .data(ImmutableMap.<String, Object>builder()
+                .put("documents_socialWorkOther", ImmutableList.of(
+                    ImmutableMap.of(
+                        "id", "12345",
+                        "value", ImmutableMap.of(
+                            "documentTitle", ""
+                        )
+                    ),
+                    ImmutableMap.of(
+                        "id", "12345",
+                        "value", ImmutableMap.of(
+                            "documentTitle", ""
+                        )
+                    )
+                )).build())
+            .build()).build();
+
+        MvcResult response = performResponseCallBack(request);
+
+        AboutToStartOrSubmitCallbackResponse callbackResponse = MAPPER.readValue(response.getResponse()
+            .getContentAsByteArray(), AboutToStartOrSubmitCallbackResponse.class);
+
+        assertThat(callbackResponse.getErrors())
```

Comment:
.contains() accepts multiple I believe, so we could assert both in one line. Can we also do something like containsExactly? or are other errors present in that response?